lang/cs/Org.Apache.REEF.Driver/Evaluator/EvaluatorRequestBuilder.cs

```diff
@@ -95,7 +103,18 @@ public EvaluatorRequestBuilder SetRackName(string rackName)
         }
 
         /// <summary>
-        /// Sets the batch ID for requested evaluators in the same request. The batch of Evaluators requested in the 
+        /// Set desired node names for the Evaluator to be allocated on.
+        /// </summary>
+        /// <param name="nodeNames"></param>
+        /// <returns>this</returns>
+        public EvaluatorRequestBuilder SetNodeNames(IList<string> nodeNames)
```

Comment:
Yes, but this is builder. The builder pattern usually only contains set methods. Not sure if we shall put different methods like Add in the builder class?